src/runtime_src/core/common/query_requests.h

```diff
@@ -387,7 +387,7 @@ struct pcie_express_lane_width_max : request
 
 struct pcie_bdf : request
 {
-  using result_type = std::tuple<uint16_t,uint16_t,uint16_t>;
+  using result_type = std::tuple<uint16_t, uint16_t, uint16_t, uint16_t>;
```

Comment:
Unfortunately, the "tuple" value is getting out-of-control for it is resulting in "magic" hard coded entry values.  Unfortunately, moving away from using this may not be possible so what minimally needs to be done done is to:
1. Document what each entry is (including a sample value).
2. Replace the hard coded index values (e.g., 0, 1, 2, etc.) with a static "string" value representing the index value.   

For example: BDF_DOMAIN, BDF_BUS, etc. where BDF_DOMAIN would have a value of 0, BDF_BUS 1, etc.

Note: Please **_do not define_** these values with define statements (e.g., #define BDF_DOMAIN 0)

Reply:
Hi @rozumx Shall I add enum for this indexing purpose in query_request.h file, like below

enum class bdf_index
{
 BDF_DOMAIN,
 BDF_BUS,
 BDF_DEV,
 BDF_FUNC,
 BDF_MAX
};